library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -584,6 +636,70 @@ public void notifyItemChangedInSection(Section section, int position) {
         callSuperNotifyItemChanged(getPositionInAdapter(section, position));
     }
 
+    /**
+     * Helper method that calculates the relative header position in the adapter and calls
+     * {@link #notifyItemChanged}. Do nothing if the section doesn't have header
+     *
+     * @param tag unique identifier of the section
+     */
+    public void notifyHeaderChangedInSection(String tag) {
+        int headerPosition = getHeaderPositionInAdapter(tag);
+
+        if (headerPosition == -1) {
+            return;
+        }
+
+        callSuperNotifyItemChanged(headerPosition);
+    }
+
+    /**
+     * Helper method that calculates the relative header position in the adapter and calls
+     * {@link #notifyItemChanged}. Do nothing if the section doesn't have header
+     *
+     * @param section a visible section of this adapter
+     */
+    public void notifyHeaderChangedInSection(Section section) {
+        int headerPosition = getHeaderPositionInAdapter(section);
+
+        if (headerPosition == -1) {
```

Comment:
this check can be removed after changing `getHeaderPositionInAdapter` to throw exception